api/controllers/v1/identities.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"github.com/bwmarrin/snowflake"
+	"github.com/gin-gonic/gin"
+	"net/http"
+	database "tasks-dispatcher/internal/durable"
+)
+
+func generate() int64{
```

Comment:
I think it's better to move the generate function to a package named for example common or utilities

Reply:
Good suggestion!